Add-ons/Umbraco-Plumber/Getting-Started/index.md

```diff
@@ -0,0 +1,37 @@
+---
+meta.Title: "Get started working with Umbraco Plumber"
+meta.Description: "Here you can find information about getting started with Umbraco Plumber"
+versionFrom: 8.0.0
+versionTo: 10.0.0
+---
+
+# Umbraco Plumber
+
+In this article, you can read about [Umbraco Plumber](https://umbraco.com/products/umbraco-plumber/) and how to get started.
+
+## Umbraco Plumber Overview
+
+Umbraco Plumber allows the creation of multi-stage approval workflows when writing and publishing content. Umbraco Plumber extends Umbraco's out-of-the-box publishing model with multi-stage, highly configurable approval workflows. A workflow process comprises multiple steps, with multiple users assigned to the group responsible for providing approval at each step.
+
+A user can be a member of multiple groups in the same workflow. To initiate an approval workflow, a user updates the content, saves their changes, then selects **Request publish**.
+
+After entering a comment describing the nature of the changes and submitting the request, members of the approving group are notified via email. The task is also displayed in the Member's Workflow dashboard. The workflow Dashboard updates to reflect the state of each task, providing an overview of a user's submissions and tasks.
+
+When initiating a workflow, the user can choose to add a document attachment, for example: an email approving or requesting the change to be made.
+
+Tasks can be approved, cancelled, or rejected from the **Dashboard** or from the content node **Workflow** tab. Rejecting a task returns the workflow to the original author, who can update the content and resubmit. The resubmitted content does not restart the workflow but returns to the stage at which it was rejected.
```

Comment:
I think this becomes a bit too much of a wall of text.

Would it be possible to shorten it? Maybe linking to some of the articles for some of the functions would make sense here? 

i think that people most likely would not read all of it when the get to this page :)

Reply:
Makes sense. Shortened the overview part.